components/camel-fory/src/generated/java/org/apache/camel/component/fory/ForyDataFormatConfigurer.java

```diff
@@ -14,20 +14,18 @@
  */
 @Generated("org.apache.camel.maven.packaging.PackageDataFormatMojo")
 @SuppressWarnings("unchecked")
-public class FuryDataFormatConfigurer extends PropertyConfigurerSupport implements GeneratedPropertyConfigurer {
+public class ForyDataFormatConfigurer extends PropertyConfigurerSupport implements GeneratedPropertyConfigurer {
 
     @Override
     public boolean configure(CamelContext camelContext, Object target, String name, Object value, boolean ignoreCase) {
-        FuryDataFormat dataformat = (FuryDataFormat) target;
+        ForyDataFormat dataformat = (ForyDataFormat) target;
         switch (ignoreCase ? name.toLowerCase() : name) {
         case "unmarshaltype":
         case "unmarshalType": dataformat.setUnmarshalType(property(camelContext, java.lang.Class.class, value)); return true;
         case "requireclassregistration":
         case "requireClassRegistration": dataformat.setRequireClassRegistration(property(camelContext, boolean.class, value)); return true;
         case "threadsafe":
         case "threadSafe": dataformat.setThreadSafe(property(camelContext, boolean.class, value)); return true;
-        case "allowautowiredfury":
```

Comment:
@oscerd apologies if I am mistaken but I couldn't find if this code has moved somewhere else but it strikes me (as a non-expert on the code) that this code may have been removed by accident - I'm happy to be corrected

Reply:
@davsclaus fixed it this morning https://github.com/apache/camel/commit/a99a112879eca22896c2599b9f36791a0b0baa5b